downstream/modules/platform/ref-eda-system-requirements.adoc

```diff
@@ -10,7 +10,10 @@ The {EDAcontroller} is a single-node system capable of handling a variable numbe
 h| Requirement | Required
 | *RAM* | 16 GB
 | *CPUs* | 4
-| *Local disk* | 40 GB minimum
+| *Local disk* a| 
+* Hard drive must be 40 GB minimum with at least 20 GB available under /var.
+* Storage volume must be rated for a minimum baseline of 1500 IOPS.
+* If the cluster has many large projects, consider doubling the GB in /var to avoid disk space errors.
```

Comment:
```suggestion
* If the cluster has many large projects or many Decision environment images, consider doubling the GB in /var to avoid disk space errors.
```